onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -397,6 +396,7 @@ struct ProviderHostImpl : ProviderHost {
 
   // logging::LoggingManager (wrapped)
   const logging::Logger& logging__LoggingManager__DefaultLogger() override { return logging::LoggingManager::DefaultLogger(); }
+  bool logging__LoggingManager__HasDefaultLogger() override { return logging::LoggingManager::HasDefaultLogger(); }
```

Comment:
This should available in the main branch now. Perhaps need to sync with latest main.